dashboard/src/components/OperatorView/OperatorDescription.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+import * as ReactMarkdown from "react-markdown";
+import OperatorDescription from "./OperatorDescription";
+
+it("renders a description", () => {
+  const wrapper = shallow(<OperatorDescription description="# Title!\n" />);
+  const markdown = wrapper.find(ReactMarkdown);
+  expect(markdown).toExist();
+  expect(markdown.html()).toContain("Title!");
```

Comment:
Without context, isn't it `<h1>Title!</h1>` or something that we should be checking here?

Reply:
well, yes, the goal of this component is not to test the `Markdown` component but it's more complete that way.